azkaban-common/src/main/java/azkaban/jobExecutor/AbstractProcessJob.java

```diff
@@ -39,6 +39,7 @@
   public static final String ENV_PREFIX = "env.";
   public static final String WORKING_DIR = "working.dir";
   public static final String JOB_PROP_ENV = "JOB_PROP_FILE";
+  public static final String JOB_PRIVATE_PROP_ENV = "JOB_PRIVATE_PROP_FILE";
```

Comment:
Consider the name _JOBTYPE_PRIVATE_PROP_ENV_.

Reply:
I was following what is there for existing files.